pom.xml

```diff
@@ -45,6 +46,11 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mapstruct</groupId>
+            <artifactId>mapstruct</artifactId>
+            <version>${org.mapstruct.version}</version>
+        </dependency>
```

Comment:
what is mapstruct libary?

Reply:
MapStruct is a Java-based code generation library used for mapping between Java bean types.

References:

- https://mapstruct.org/documentation/reference-guide/
- https://www.baeldung.com/mapstruct

Reply:
Ok! I'm understand, thank you!